src/Tests/AddThisFieldTest.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * @file
+ * Definition of Drupal\addthis\Tests\AddThisFieldTest.
+ */
+namespace Drupal\addthis\Tests;
+
+/**
+ * Tests the basic functionality provided by AddThis.
+ *
+ * @group addthis
+ */
+class AddThisFieldTest extends AddThisFieldWebTestBase {
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = array('addthis', 'node', 'field', 'dblog');
+
+  /**
+   * User account with all available permissions
+   *
+   * @var \Drupal\Core\Session\AccountInterface
+   */
+  protected $adminUser;
+
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(){
+    parent::setUp();
+    $this->adminUser = $this->drupalCreateUser(array_keys(\Drupal::service('user.permissions')->getPermissions()));
```

Comment:
Probably a good idea to narrow down the permissions set. I can take care of that today